perses/annihilation/new_relative.py

```diff
@@ -70,6 +75,12 @@ def __init__(self, topology_proposal, current_positions, new_positions, use_disp
         self.softcore_alpha=0.5
         self.softcore_beta=12*unit.angstrom**2
```

Comment:
Are you sure you want to use the fused softcore here, where the charges are softened at intermediate lambda, or did you just want to linearly change charges with `lambda`?